streams/src/test/java/org/apache/kafka/streams/integration/IQv2VersionedStoreIntegrationTest.java

```diff
@@ -88,7 +87,7 @@ public class IQv2VersionedStoreIntegrationTest {
 
     public static final EmbeddedKafkaCluster CLUSTER = new EmbeddedKafkaCluster(NUM_BROKERS, Utils.mkProperties(Collections.singletonMap("auto.create.topics.enable", "true")));
 
-    private KafkaStreams kafkaStreams;
+    private static KafkaStreams kafkaStreams;
```

Comment:
Not sure if it's ok to make this one `static`? It's setup inside `@Before` which is not static and we run test in parallel...

I think we need to rather pass it as a parameter into the newly added `static` methods that use it.